packages/run-protocol/src/vaultFactory/vaultManager.js

```diff
@@ -316,7 +321,9 @@ const helperBehavior = {
       prioritizedVaults.entriesPrioritizedGTE(quoteRatioPlusMargin);
     await (next ? facets.helper.liquidateAndRemove(next) : null);
 
-    facets.helper.reschedulePriceCheck();
+    trace('price check liq', next && next[0]);
+
+    await facets.helper.reschedulePriceCheck();
```

Comment:
would this get more execution into the same turn?
```suggestion
    return facets.helper.reschedulePriceCheck();
```

Reply:
It complains if you try to return something because of "inconsistent return". Since they are in theory equivalent, it's not worth working up.

Reply:
> they are in theory equivalent

I thought the `await` would force something to pause until the next turn.

Also, won't awaiting a recursive call cause a monotonically growing call stack? 

>It complains if you try to return something because of "inconsistent return"

Ah, that's eslint `consistent-return` complaining because it doesn't know the real type. In both cases the return should be `Promise<void>` so the eslint erroneous error is fair to suppress.

Reply:
Ok that is a fair case of eslint being too limited not knowing the return type.

That said, for stack traces reason, I believe the `await` version is preferable.